kw

```diff
@@ -204,6 +204,13 @@ function kw()
         pomodoro_main "$@"
       )
       ;;
+    profiler)
+      (
+        include "${KW_LIB_DIR}/profiler.sh"
+
+        profiler_main "$@"
+      )
+      ;;
```

Comment:
If you try my suggestion from the previous commit, you will probably need to rethink this part. I see two possible paths:
1. You add some sort of "guards" around this code that only include this if the `--enable-trace` is used.
2. You create a new tool named `profile` under a folder named `tools/profile/`

Reply:
Done